diff -up gexiv2-0.10.9/gexiv2/gexiv2-metadata.cpp.omv~ gexiv2-0.10.9/gexiv2/gexiv2-metadata.cpp
--- gexiv2-0.10.9/gexiv2/gexiv2-metadata.cpp.omv~	2018-12-24 02:23:56.016024918 +0100
+++ gexiv2-0.10.9/gexiv2/gexiv2-metadata.cpp	2018-12-24 02:24:00.124088321 +0100
@@ -26,7 +26,6 @@
 #include <exiv2/xmpsidecar.hpp>
 #include <exiv2/exif.hpp>
 #include <exiv2/iptc.hpp>
-#include <exiv2/xmp.hpp>
 #include <exiv2/preview.hpp>
 #include <exiv2/jpgimage.hpp>
 
diff -up gexiv2-0.10.9/gexiv2/gexiv2-metadata-private.h.omv~ gexiv2-0.10.9/gexiv2/gexiv2-metadata-private.h
--- gexiv2-0.10.9/gexiv2/gexiv2-metadata-private.h.omv~	2018-12-24 02:23:43.297828499 +0100
+++ gexiv2-0.10.9/gexiv2/gexiv2-metadata-private.h	2018-12-24 02:23:48.061902098 +0100
@@ -14,7 +14,7 @@
 #include <exiv2/image.hpp>
 #include <exiv2/exif.hpp>
 #include <exiv2/iptc.hpp>
-#include <exiv2/xmp.hpp>
+#include <exiv2/xmp_exiv2.hpp>
 #include <exiv2/preview.hpp>
 
 G_BEGIN_DECLS
diff -up gexiv2-0.10.9/gexiv2/gexiv2-metadata-xmp.cpp.omv~ gexiv2-0.10.9/gexiv2/gexiv2-metadata-xmp.cpp
--- gexiv2-0.10.9/gexiv2/gexiv2-metadata-xmp.cpp.omv~	2018-12-24 02:24:15.059318666 +0100
+++ gexiv2-0.10.9/gexiv2/gexiv2-metadata-xmp.cpp	2018-12-24 02:24:17.170351203 +0100
@@ -12,7 +12,6 @@
 #include "gexiv2-metadata-private.h"
 #include <string>
 #include <glib-object.h>
-#include <exiv2/xmp.hpp>
 
 G_BEGIN_DECLS
 
diff -up gexiv2-0.10.9/gexiv2/gexiv2-startup.cpp.omv~ gexiv2-0.10.9/gexiv2/gexiv2-startup.cpp
--- gexiv2-0.10.9/gexiv2/gexiv2-startup.cpp.omv~	2018-12-24 02:24:26.616496739 +0100
+++ gexiv2-0.10.9/gexiv2/gexiv2-startup.cpp	2018-12-24 02:24:54.524926159 +0100
@@ -7,7 +7,7 @@
  * This is free software. See COPYING for details.
  */
 
-#include <exiv2/xmp.hpp>
+#include <exiv2/xmp_exiv2.hpp>
 #include "gexiv2-startup.h"
 #include "gexiv2-version.h"
 
